docs/README.md

```diff
@@ -173,7 +175,9 @@ After running the `build_docker.sh` the following docker images will be availabl
 Now the "Media Proxy" can be run inside the container. To check it, execute bellow command:
 
 ```bash
-$ docker run --privileged -it -v /var/run/mcm:/run/mcm -v /dev/hugepages:/dev/hugepages mcm/media-proxy:latest
+docker run --privileged -it -v /var/run/mcm:/run/mcm -v /dev/hugepages:/dev/hugepages mcm/media-proxy:latest
+```
+```
```

Comment:
I would add here (before INFO: TCP Server..." the line. sth like:
The result should look like this:

Reply:
Fixed.